README.md

```diff
@@ -6,19 +6,22 @@
 
 <!--
     Copyright (c) 2014, Joyent, Inc.
+    Copyright 2022 MNX Cloud, Inc.
 -->
 
 # node-mahi
 
-This repository is part of the Joyent SmartDataCenter project (SDC).  For
-contribution guidelines, issues, and general documentation, visit the main
-[SDC](http://github.com/joyent/sdc) project page.
+This repository is part of Triton Data Center.  For contribution
+guidelines, issues, and general documentation, visit the main
+[Triton](http://github.com/TritonDataCenter/triton) project page.
 
 This is the client for Mahi. When talking to mahi, translation and
 authentication responses are cached. node-mahi also contains the authorization
 API.
 
 # Testing
 
-Testing requires a Mahi server instance. Point MAHI_TEST_URL at the test
-instance and run `make test`.
+Testing requires a Mahi server instance.  Point MAHI\_TEST\_URL at an existing
```

Comment:
Should be `MAHI_TEST_URL`.

I.e.:

    `MAHI_TEST_URL`

not

    MAHI\_TEST\_URL